apps/web/src/app/api/commits/[commitUuid]/documents/route.ts

```diff
@@ -0,0 +1,45 @@
+import materializeDocumentsAtCommit from '$/services/commitService/materializeDocumentsAtCommit'
+import { DocumentType } from '$core/schema'
```

Comment:
I think I fixed this issue of importing with the internal path
![image](https://github.com/user-attachments/assets/6f606c7f-ac0e-4592-864d-ddd56f191b7c)

Add this to `web/tsconfig.json` and let me know if it fix it
![image](https://github.com/user-attachments/assets/0da09a13-aa52-47f4-bc00-65a4774b0809)

Reply:
hmm keep having the issue